contracts/IERCBlob.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IERCBlob {
+
+    function writeChunkBlob(
```

Comment:
To simplify the logic, we can define the chunk size is the same as the blob size. So the interface can be: function writeChunk(bytes memory name, uint256[] chunkIds)